src/Microsoft.PowerShell.Commands.Utility/commands/utility/GetDateCommand.cs

```diff
@@ -369,9 +369,7 @@ private string UFormatDateString(DateTime dateTime)
                             break;
 
                         case 'c':
-                            sb.Append("{0:ddd} {0:MMM} ");
-                            sb.Append(StringUtil.Format("{0,2} ", dateTime.Day));
-                            sb.Append("{0:HH}:{0:mm}:{0:ss} {0:yyyy}");
+                            sb.Append("{0:ddd} {0:MMM} {0:dd} {0:HH}:{0:mm}:{0:ss} {0:yyyy}");
```

Comment:
Apparently, there are platform differences (not specific to the `date` _utility_ implementation): on macOS 10.12.6 I get space-padded representation (e.g., `<space>7`), on Ubuntu 16.04 I get the `0`-padded representation (e.g., `07`).

Reply:
On WSL I get space too. And it seems it can be culture-depended (see related Issue). I think we can not be full-compliant with Unix here. So we should follow "common" rule.

Reply:
> we should follow "common" rule.

I'm asking innocently: what do you mean by that?

I just checked the [spec for the POSIX `date` utility](http://pubs.opengroup.org/onlinepubs/9699919799/utilities/date.html), and the definition of `%c` is (a) definitely locale-(culture)-specific and (b) pretty open-ended:

> %c
> Locale's appropriate date and time representation.
> ...
> The %x and %c conversion specifications, however, are intended for local representation; these may be based on a different, non-Gregorian calendar.

Reply:
Sorry, I meant that "common" - used in most Unix clones.
Thank you for confirmation this is a locale-(culture)-specific. The problem may lie in the fact that we are forced to rely on CoreFX rather than Unix locales. I think that's a good compromise because our goal is to _emulate_ Unix-like format. If we want more we should address this in CoreFX.